lesson26/src/login.html

```diff
@@ -13,11 +13,12 @@
           <h1>ログイン</h1>
           <div class="mb-6">
             <label for="username">お名前 / メールアドレス<span class="required text-red-500 mx-1" aria-hidden="true">*</span></label>
-            <input type="text" name="username" id="username" placeholder="山田　太郎 / sample@sample.com" required>
+            <input type="text" name="username" id="username" autocomplete="username" placeholder="山田　太郎 / sample@sample.com" required>
           </div>
-          <div class="mb-6">
+          <div class="mb-6 relative">
             <label for="confirm-password">パスワード<span class="required text-red-500 mx-1" aria-hidden="true">*</span></label>
-            <input type="password" name="confirm-password" id="confirm-password" autocomplete="confirm-password" placeholder="********" required>
+            <button id="toggle-password" type="button" class="absolute cursor-pointer py-1 h-[42px] px-2 top-[32px] right-0 text-[16px] rounded-sm font-bold text-blue-500" aria-label="パスワードを表示">表示</button>
+            <input type="password" name="current-password" id="current-password" autocomplete="current-password" placeholder="********" aria-describedby="password-constraints" required>
```

Comment:
nits 
I thought it would be better to have the js-prefix in id if it did not interfere with other implementations.

Reply:
Thank you for your comment!
I'll modify the `id="toggle-password"` to add `js-`.